django/chat/templates/chat/modals/presets/presets_form.html

```diff
@@ -115,7 +116,10 @@
                   hx-post="{% if preset_id %}{% url 'chat:chat_options' chat_id=chat_id action='save_preset' preset_id=preset_id %}{% else %}{% url 'chat:chat_options' chat_id=chat_id action='save_preset' %}{% endif %}"
                   hx-include="#edit_preset_form"
                   hx-target="#presets-modal-body"
-                  hx-swap="innerHTML">{% trans "Save" %}</button>
+                  hx-swap="innerHTML"
+                  onclick="document.querySelector('input[name=\'prompt\']').value = document.getElementById('chat-prompt').value">
```

Comment:
Just before the Preset form submit, the chat prompt value is copied into the hidden field.